packages/web/src/components/basic/ReactiveComponent.js

```diff
@@ -108,8 +108,24 @@ class ReactiveComponent extends Component {
 			filterLabel,
 			showFilter,
 			URLParams,
+			aggregationField,
+			config,
+			distinctField,
+			distinctFieldConfig,
 		} = this.props;
 		const initialValue = selectedValue || value || defaultValue || null;
+		const { enableAppbase } = config;
+
+		if (aggregationField !== undefined) {
```

Comment:
@anik-ghosh-au7 Display this warning message only when `enableAppbase` is set to `true`. Make that change everywhere.

Reply:
@bietkul I have updated the changes